data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1066,6 +1066,99 @@ ship "Centipede" "Centipede (Teeneep Jump)"
 
 
 
+ship "Frugal Defense Cannon"
+	attributes
+		"heat dissipation" 1
+		"hull threshold" 200
+		# Once this loses 50 hull ((400 - 200) / 4 hull = 50), it will stop being frugal.
+		# Since the gun does 0.5 hull damage per second, this will take exactly 100 seconds.
+		# The total time it takes to make this switch is also important to be sure that the gun is properly in position, too.
+		"hull" 400
+		"mass" 25000
+		"drag" 25
+		"automaton" 1
+		"landing speed" 1
+	outfits
+		"Timer Weapon Rangeless"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
```

Comment:
This could be collapsed to a single line, I don't understand why it's expanded out like this?
```suggestion
		"Defense Cannon" 18
```